README.md

```diff
@@ -89,6 +89,12 @@ To Start a Docker Compose Instance for Testing / Debugging
     
    You can start multiple compose instances on the same project as the plugin generates a unique name for each instance.
    
+   To use the static host ports instead of the Docker dynamically assigned host ports use the following command:
```

Comment:
We should add some extra text here about why the feature is useful. Something like:

> When making frequent code changes on your local machine it is often useful to temporarily have the external ports remain the same. The -useStaticPorts argument provide this functionality.